### ISSUE TITLE
Hurt Notes

### ISSUE THREAD

**Issue by Joacx21** [JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag#254]
### Describe your bug here. Mind you, if you are modding, please try testing your bug in a clean version of the engine instead, this ensures the bug is actually caused by the engine itself. Also, if you're using an older version of JSE, please try the latest action build in the Actions tab or the latest release build. The issue has probably been fixed since that version.

For some reason, when i place hurt notes in a chart and then play it, they behave like normal notes, they don't hurt the player, it's really weird

### Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.

```bash
I just place the notes in the chart editor, test the chart, and it doesn't matter how i access the chart, since it always happens
```

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

Basically was making a mod, discovered the bug, then double check with a unmodified build and the bug was still there

### If you use 1.12.0 or earlier, did you have Optimized Chart Loading turned on?

None

### Did you check for any similar issues to what you're reporting? Check, then come back here. If there is a similar issue, then do not report the issue, otherwise it will be marked as a duplicate.

Yes

**Comment by JordanSantiagoYT** [comment]
Hey there, I'm actually already aware of this and am working on a fix.

**Comment by Joacx21** [comment]
thanks.
